openapi/index.yaml

```diff
@@ -357,6 +357,76 @@ paths:
           description: Too many requests
       parameters:
       - $ref: "#/parameters/DateUTC"
+  "/services":
+    post:
+      operationId: createService
+      summary: Create Service
+      description: |
+        Create a new Service with the attributes provided in the request payload.
+      parameters:
+        - in: body
+          name: body
+          description: The service payload.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
```

Comment:
What if subscription creation fail?

Reply:
Add 500

Reply:
fixed in [
a9bab54](https://github.com/pagopa/io-functions-services/pull/67/commits/a9bab54907fcb5c5e8e09c2941ee4fe0e103bd85)